xbmc/guilib/GUIFontManager.cpp

```diff
@@ -507,10 +507,7 @@ void GUIFontManager::LoadFonts(const TiXmlNode* fontNode)
 
     if (!fontName.empty() && URIUtils::HasExtension(fileName, ".ttf"))
     {
-      //! @todo Why do we tolower() this shit?
-      std::string strFontFileName = fileName;
-      StringUtils::ToLower(strFontFileName);
-      LoadTTF(fontName, strFontFileName, textColor, shadowColor, iSize, iStyle, false, lineSpacing,
+      LoadTTF(fontName, fileName, textColor, shadowColor, iSize, iStyle, false, lineSpacing,
```

Comment:
Looks much better as this 👍. Was also confused where seen this place in the paste.

If any problem will comes again after the request comes in, we should look on right place and no more the hacks 😆.

Reply:
note for future self: if there is a real problem with the font filename case then fixes could be to not cache the font paths, or to add a case-insensitive cache lookup (or maybe it should always be case-insensitive).